core/chains/evm/chain_set.go

```diff
@@ -361,11 +361,20 @@ func LoadChainSet(opts ChainSetOpts) (ChainSet, error) {
 	if err := checkOpts(&opts); err != nil {
 		return nil, err
 	}
-	dbchains, nodes, err := opts.ORM.EnabledChainsWithNodes()
+	chains, err := opts.ORM.EnabledChains()
```

Comment:
Inlining this single usage since `EnabledChainsWithNodes()` was a bit of an oddity.